src/daemon.rs

```diff
@@ -506,13 +538,12 @@ impl Daemon {
             .iter()
             .map(|txhash| json!([txhash.to_hex(), /*verbose=*/ false]))
             .collect();
-
-        let values = self.requests("getrawtransaction", &params_list)?;
+        let values = self.retry_request_batch("getrawtransaction", &params_list, 0.25)?;
```

Comment:
1. Magic number / arbitrary value.
2. This doesn't feel like it should be set here.

At the very least we need a TODO to properly bubble this up the call stack and make it into a startup config param or something.